debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/SourceInfo.java

```diff
@@ -26,7 +26,7 @@ public class SourceInfo extends BaseSourceInfo {
     public static final String SNAPSHOT_KEY = "snapshot";
```

Comment:
I wonder if source info schema should change too and use array of strings.

Reply:
So the string-based implementation just writes the complete contents of the "commit_scn" value-set to the source info field however, thinking about this more, I don't believe that's exactly what we want.  

A single transaction is only ever associated with a given Oracle RAC node, and therefore the source info block should only refer to a single commit SCN data point, which is the node the transaction happened on.  I'll adjust that behavior first in the string-based implementation by tracking the Oracle RAC node's redo thread in `SourceInfo`.

Once that is done, the schema change would in fact simply be an array of fields as you described.